python/docker-compose.python.yml

```diff
@@ -179,25 +116,24 @@ services:
             # Direct connection to DB
             DB_HOST: postgresql
             DB_PORT: 5432
-            # Connectin through AcraConnector
-            ACRA_CLIENT_ID: ${ACRA_CLIENT_ID:-testclientid}
-            ACRA_CONNECTOR_HOST: acra-connector
-            ACRA_CONNECTOR_PORT: 9494
-            # Demo script default cofiguration
-            EXAMPLE_ACRA_CONNECTOR_API_ADDRESS: http://acra-connector:9191
-            EXAMPLE_HOST: acra-connector
-            EXAMPLE_PORT: 9494
-            EXAMPLE_DB_USER: ${POSTGRES_USER:-test}
+            # Demo script default configuration
+            EXAMPLE_ACRA_CONNECTOR_API_ADDRESS: http://acra-server:9090
```

Comment:
should it be named `..acra_connector..`?

Reply:
Yes, need to fix it in acra examples firstly.